lib/PrimitiveHelpers.js

```diff
@@ -163,6 +186,14 @@ function transformPrimitives(gltf, primitives, transform) {
     }
 }
 
+/**
+ * Get a hash table like object with the material as the hash and the
```

Comment:
There are actually two levels to this, material and mode (e.g. WebGLConstants.TRIANGLES). Also, just call them objects, referring to hash tables here is redundant.